softgroup/basis/common-model-mapper/src/test/java/com/softgroup/common/model/mapper/ModelMapperTest.java

```diff
@@ -0,0 +1,105 @@
+    }
+
+    @Test
+    public void fromPerson1ToPersonDTOmapMethodTest(){
+        personDTO = modelMapper.map(person1, PersonDTO.class);
+
+        assertThat(personDTO, notNullValue());
+        assertThat(personDTO.getName(), is(person1.getName()));
+        assertThat(personDTO.getLastName(), is(person1.getLastName()));
+        assertThat(personDTO.getAge(), is(person1.getAge()));
+        assertThat(personDTO.getCity(), is(person1.getAddress().getCity()));
+        assertThat(personDTO.getStreet(), is(person1.getAddress().getStreet()));
+        assertThat(personDTO.getCountry(), is(person1.getAddress().getCountry()));
+
+        Address mappedAddress = modelMapper.map(personDTO, Address.class);
+
+        assertThat(mappedAddress, notNullValue());
+        assertThat(mappedAddress.getCity(), is(personDTO.getCity()));
+        assertThat(mappedAddress.getStreet(), is(personDTO.getStreet()));
+        assertThat(mappedAddress.getCountry(), is(personDTO.getCountry()));
+
+        Person mappedPerson = modelMapper.map(personDTO,Person.class);
+
+        assertThat(mappedPerson, notNullValue());
+        assertThat(mappedPerson.getName(), is(personDTO.getName()));
+        assertThat(mappedPerson.getLastName(), is(personDTO.getLastName()));
+        assertThat(mappedPerson.getAge(), is(personDTO.getAge()));
+        assertThat(mappedPerson.getAddress().getCity(), is(personDTO.getCity()));
+        assertThat(mappedPerson.getAddress().getStreet(), is(personDTO.getStreet()));
+        assertThat(mappedPerson.getAddress().getCountry(), is(personDTO.getCountry()));
+    }
+
+    @Test
+    public void fromPerson2ToPersonDTOmapMethodTest(){
+        personDTO = modelMapper.map(person2, PersonDTO.class);
+
+        assertThat(personDTO, notNullValue());
+        assertThat(personDTO.getName(), is(person2.getName()));
+        assertThat(personDTO.getLastName(), is(person2.getLastName()));
+        assertThat(personDTO.getAge(), is(person2.getAge()));
+        assertThat(personDTO.getCity(),  nullValue());
+        assertThat(personDTO.getStreet(), nullValue());
+        assertThat(personDTO.getCountry(), nullValue());
+
+        Address mappedAddress = modelMapper.map(personDTO, Address.class);
+
+        assertThat(mappedAddress, notNullValue());
+        assertThat(mappedAddress.getCity(), nullValue());
+        assertThat(mappedAddress.getStreet(), nullValue());
+        assertThat(mappedAddress.getCountry(), nullValue());
+
+        Person mappedPerson = modelMapper.map(personDTO,Person.class);
+
+        assertThat(mappedPerson, notNullValue());
+        assertThat(mappedPerson.getName(), is(personDTO.getName()));
+        assertThat(mappedPerson.getLastName(), nullValue());
+        assertThat(mappedPerson.getAge(), is(personDTO.getAge()));
+        assertThat(mappedPerson.getAddress(), nullValue());
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void mapWithWrongArgumentsTest1(){
```

Comment:
@AlexKazmirchuk Use proper names. 1, 2 etc. is not proper names as they did not provide any information.